package/src/utils/utils.ts

```diff
@@ -2,17 +2,16 @@ import type React from 'react';
 
 import dayjs from 'dayjs';
 import EmojiRegex from 'emoji-regex';
-import type { DebouncedFunc } from 'lodash';
+import { DebouncedFunc } from 'lodash';
```

Comment:
Let's revert this one. I'm not sure if this change will lead to bundling the whole `lodash` library.